ShoppingStore/Store.java

```diff
@@ -0,0 +1,81 @@
+import java.util.ArrayList;
+import java.util.List;
+
+public class Store {
+	private List<Product> products = new ArrayList<Product>();
+
+	public Store() {
+		this.initStoreItems();
+	}
+
+	/**
+	 * Read from product file and initializing product class object and add to
+	 * products List
+	 */
+	public void initStoreItems() {
+		List<String> productsDetails = new FileHandling()
+				.read(Constants.PRODUCT_LIST_PATH);
+		for (String str : productsDetails) {
+			String[] details = str.split(",");
```

Comment:
Deserialize logic can be moved inside product class, as a new constructor, which takes serialized string as input